### DISCUSSION TITLE
Embedding on self-hosted

### DISCUSSION THREAD

**Discussion by Kasper-B** [lightdash/lightdash#12486] [Q&A] [answered]
How to i enable the embedding feature on self-hosted setup of Lightdash?

**Comment by jpetey75** [comment, marked as answer]
Hey @Kasper-B right now embedding is only offered on [paid plans](https://www.lightdash.com/pricing) of Lightdash. Right now it only works for Cloud customers, but we're working on an option for paid self-hosted customers to use embedding as well. That will be ready soon.

If you're interested in learning more you can [reach out here](https://lightdash.typeform.com/to/BujU5wg5) and we'll be in touch!
